### ISSUE TITLE
Parse Precinct-Level Results for 2016 General Election

### ISSUE THREAD

**Issue by dwillis** [openelections/openelections-data-nj#26]
Using [Tabula](http://tabula.technology/), parse precinct-level results for the following counties. Original sources are in the [sources-nj repository](https://github.com/openelections/openelections-sources-nj/tree/master/2016).

The goal is to create a single CSV file for each county, with the following headers:

`county`, `precinct`, `office`, `district`, `party`, `candidate`, `votes`

For the following offices: President, U.S. House, State House (there were two special elections for this office, so most counties will not have any state house races). You can use the [Essex County](https://github.com/openelections/openelections-data-nj/blob/master/2016/20161108__nj__general__essex__precinct.csv) file as an example of how things should look. You do not need to include municipal totals.

- [x] Atlantic
- [x] Bergen
- [x] Burlington
- [x] Camden
- [x] Cape May
- [x] Cumberland
- [x] Essex
- [x] Gloucester
- [x] Hudson
- [x] Hunterdon
- [x] Mercer
- [x] Middlesex
- [x] Monmouth
- [x] Morris
- [x] Ocean
- [x] Passaic
- [x] Salem
- [x] Somerset
- [x] Sussex
- [x] Union
- [x] Warren

**Comment by cholq** [comment]
I'll work on Sussex

**Comment by xangregg** [comment]
Trying Hudson.

**Comment by cholq** [comment]
I'll work on Hunterdon

**Comment by srge1827** [comment]
I'll try Bergen

**Comment by srge1827** [comment]
For Warren County, minor candidates have "Nomination by Petition" as their party. Should I use that or the party names from other counties?

**Comment by dwillis** [comment]
@srge1827 you can just skip the party value for folks listed as "nomination by petition".

**Comment by dwillis** [comment]
@srge1827 I've added the Warren congressional results to the `sources` repo - they are Word docs!

**Comment by rafazaya** [comment]
I'll grab Burlington - is there a good repo into which I should merge the results file?

**Comment by dwillis** [comment]
And I've got Camden.

**Comment by rafazaya** [comment]
So Burlington is more challenging than i'd anticipated - the underlying data in the pdf is not text-based, so tabula and some other tools I threw at this did not work automatically.

However, some googling lead me to the Burlington's [Public Records Electronic Search System (PRESS) datastore](http://press.co.burlington.nj.us/PRESS/clerk/Election.aspx?op=ELE) of election results, but those are by municipality, not precinct.  I can parse those results - would that work?

**Comment by srge1827** [comment]
I'll finish up the Warren results that we have for 5th and 7th CD and then update when we get the rest of the precincts.

**Comment by dwillis** [comment]
Hey @rafazaya - yeah, we'll be glad to have results by municipality, too. Thanks!

**Comment by cholq** [comment]
@dwillis - Doesn't the file 20161108__nj__general__municipal.csv already have all NJ municipal data for 2016?  It is for all municipalities in all counties, but it should be there already.  

@rafazaya - I had volunteered to do the precinct file for Hunterdon, but I've made very little headway and have some other things going on right now.  If you are really looking for something, I can hand off Hunterdon off to you.
